src/runtime/components/forms/SelectMenu.vue

```diff
@@ -440,11 +451,16 @@ export default defineComponent({
     })
 
     const trailingIconClass = computed(() => {
+      let clearableClass = ''
+      if (props.clearable && (Array.isArray(props.modelValue) ? props.modelValue.length : props.modelValue)) {
+        clearableClass = 'hover:bg-gray-500 dark:hover:bg-gray-400 cursor-pointer pointer-events-auto'
```

Comment:
This should come from the theme config and not hard-coded in the component using `ui.value.clearableIcon` for example.